scripts/glean.py

```diff
@@ -188,11 +190,16 @@ def get_dependencies(self):
 
     def get_metrics(self) -> List[GleanMetric]:
         data = _cache.get_json(GleanApp.METRICS_URL_TEMPLATE.format(self.app["v1_name"]))
-        metrics = list(data.items())
-
+        metrics = [(m[0], {**m[1], "namespace": self.app["app_name"]}) for m in data.items()]
```

Comment:
I'm using "namespace" as the key for where the metric definition lies in the glean-annotations repository. Not sure if this is the best name (I think we use "namespace" elsewhere for different things related to probe metadata), but it was the best I could come up with (this JSON produced by the glean dictionary isn't meant to be used by anyone else).

Reply:
How about "origin" or "metric_origin" or "defined_in"?

Reply:
Origin sounds good, let's try that. `defined_in` is used for defining the line numbers in source control in probe scraper, I think we should not use it here to avoid confusion.